README.md

```diff
@@ -1,4 +1,4 @@
-# Monitoring Role
+# Monitoring Role ![Ansible Role](https://img.shields.io/ansible/role/44192)
```

Comment:
Would be nice to point the URL to https://galaxy.ansible.com/coopdevs/monitoring_role